docs/source/settings.rst

```diff
@@ -38,10 +38,27 @@ SEND_ACTIVATION_EMAIL
 If ``True`` user will be required to click activation link sent in email after:
 
 * creating an account via ``RegistrationView``
-* updating his email via ``UserView``
+* updating ``User.EMAIL_FIELD`` via ``/users/me/`` endpoint (also depends on ``ACTIVATION_NEEDED_ON_EMAIL_UPDATE`` setting)
+* updating ``User.USERNAME_FIELD`` via ``/users/change_username/`` endpoint (also depends on ``ACTIVATION_NEEDED_ON_USERNAME_UPDATE`` setting)
```

Comment:
```suggestion
* updating ``LOGIN_FIELD`` via ``/users/change_username/`` endpoint (also depends on ``ACTIVATION_NEEDED_ON_USERNAME_UPDATE`` setting)
```
``Set Username`` documentation in ``base_endpoint`` only talks about ``User.USERNAME_FIELD`` but by looking at the code, it seems like it's ``LOGIN_FIELD`` that is edited instead?